src/components/MyNotifications/MyNotifications.tsx

```diff
@@ -105,6 +105,7 @@ export default function MyNotifications({
                 {(!!item.link || !!item.customLink) && (
                   <p className={styles.Action}>
                     <a
+                      aria-label="Meer informatie over deze melding"
```

Comment:
I think this label should be used as the link.title. The textual context remains unclear for the reader. The aria-label is only for visually impaired users.

Reply:
Done